build_tools/cmake/iree_alwayslink.cmake

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Additional libraries containing statically registered functions/flags, which
+# should always be linked in to binaries.
+
+set(_ALWAYSLINK_LIBS
+  MLIRAffineOps
+  MLIRAnalysis
+  MLIREDSC
+  MLIRIR
+  MLIRParser
+  MLIRPass
+  MLIRSPIRV
+  MLIRSPIRVSerialization
+  MLIRSPIRVTransforms
+  MLIRStandardOps
+  MLIRTransforms
+  MLIRTranslation
+  MLIRSupport
+  MLIRVectorOps
+  MLIRLinalgOps
+  tensorflow::mlir_xla
+)
+
+foreach(LIB ${_ALWAYSLINK_LIBS})
+# Aliased targets are always in-project, so we control them and can set
```

Comment:
Can remove this comment and possibly the `get_target_property(... ALIASED_TARGET)` branching?
It looks like there are no aliased targets in the list now.

Reply:
Done :)

Reply:
:+1: